lexicon/conversion.py

```diff
@@ -140,6 +140,118 @@ def run(self):
             lexicon_file.write(ET.tostring(root, "unicode"))
 
 
+class LexiconFromTextFileJob(Job):
+    """
+    Create a bliss lexicon from space or tab separated files.
```

Comment:
Is this maybe the same as https://github.com/rwth-i6/i6_core/blob/cef8a57cf0983b810c85c8f5507cc05a34ba4d19/g2p/convert.py#L44

See one example g2p Lexicon here:
/work/asr4/michel/setups-data/librispeech/g2p/convert/BlissLexiconToG2PLexiconJob.AFMZQsJUiOIA/output/g2p.lexicon

Reply:
Ah no, it is definitely different!